docker-compose.yml

```diff
@@ -0,0 +1 @@
+version: "3.4"
```

Comment:
That's all? :)

Reply:
@AlexeyDemedetskiy any useful links to how to start with Docker?

Reply:
More will come in next one. This is just folders for 2 projects.